graphql/e2e/common/common.go

```diff
@@ -70,6 +70,9 @@ var (
 		"indicating that the GraphQL layer didn't get the updated schema even after 10" +
 		" retries. The most probable cause is the new GraphQL schema is same as the old" +
 		" GraphQL schema."
+
+	customAdminURL string
+	namespaceCount = 1
```

Comment:
you can define this inside the test/function itself

Reply:
Resolved